drivers/net/slip.c

```diff
@@ -30,8 +30,12 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include <net/net_pkt.h>
```

Comment:
Please change the commit subject a bit, so `net: slip: ....`

Reply:
done